pkg/graphviz/traceflow_test.go

```diff
@@ -181,6 +187,216 @@ func TestGenGraph(t *testing.T) {
 		},
 	}
 
+	liveTfWithSourceIPTwoNodes := crdv1alpha1.Traceflow{
```

Comment:
@urharshitha please check and fix the github check failures, here it shows `liveTfWithSourceIPTwoNodes declared but not used (typecheck)`